.github/workflows/deploy.yaml

```
@@ -54,5 +54,5 @@ jobs:
         uses: ./.github/actions/send-slack-message
         if: ${{ failure() }}
         with:
-          slack_webhook_url: ${{ vars.SLACK_WEBHOOK_URL }}
+          slack_webhook_url: ${{ secrets.SLACK_WEBHOOK_URL }}
```

actionlint (1.7.7)

property "slack_webhook_url" is not defined in object type {actions_runner_debug: string; actions_step_debug: string; admin_github_token: string; aws_account_id_development: string; aws_account_id_production: string; github_token: string}

(expression)